packages/integration-sdk-entity-validator/src/entityValidationError.ts

```diff
@@ -0,0 +1,38 @@
+import { ErrorObject } from 'ajv';
+
+export const unknownPropertySymbol = Symbol('_unknown_');
+
+export type EntityValidationError = {
+  property: string | typeof unknownPropertySymbol;
+  message: string;
+  validation: string;
+};
+
+const getPropertyNameFromAjvError = (
+  error: ErrorObject,
+): EntityValidationError['property'] => {
+  // instancePath is either '/{propertyName}', or if it's a list: '/{propertyName}/index'
+  // instancePath will always start with a slash
+  return error.instancePath.split('/')[1] || unknownPropertySymbol;
```

Comment:
"instancePath will always start with a slash" does that also apply even if the error is an object-level error?

I think it does, but I can't be certain.

Because if there's ever an error where `instancePath == ""`, grabbing the index 1 of the split() will throw an error.

Reply:
JS don't throw exceptions to "invalid" index because all arrays have "Infinite" size.
![image](https://github.com/JupiterOne/sdk/assets/4943174/60abdb26-7b6f-4b0f-800e-16b912f4aa38)